lib/blockchain.rb

```diff
@@ -31,4 +32,9 @@ def hash_transaction(data, time)
     sha.update(data['sender'] + data['receiver'] + time.to_s)
     sha.hexdigest
   end
+
+  def invalid?(new_block)
+    current_block = @blocks[@blocks.length - 1]
```

Comment:
• Blocks shouldn't need an @ as it's covered by an attr_reader (I might be wrong here?)
• blocks.last might be a neater way to get the previous block

Reply:
you are right, bad habit :)